cognite/client/data_classes/_base.py

```diff
@@ -764,7 +758,7 @@ def load(
                 nulls=data[2],
             )
         elif isinstance(data, str) and (prop_order := data.split(":", 1))[-1] in ("asc", "desc"):
-            # Syntax "<fieldname>:asc|desc" is depreacted but handled for compatibility
+            # Syntax "<fieldname>:asc|desc" is deprecated but handled for compatibility
```

Comment:
Clearly none of us core maintainers can read

Reply:
I'll add a spellchecker to your linters for you 🤣

Reply:
Yes, please 🚀